### PR TITLE
Math Magicians: Events

### PR DESCRIPTION
**Pull request by rmiyoyo**
# Work Done
- Created a new directory called logic inside my project.
- Copied the files ```calculate.js``` and ```operate.js``` into my logic/ directory.
- Installed [```big.js```](https://www.npmjs.com/package/big.js) using NPM.
- Implemented click event handlers using the math logic from calculate.js and operate.js in my React component.
- The Calculator is able to perform math operations.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
```
Comment (rloterh) [inline src/logic/operate.js:20]: - [ ] When I divide a number by `0` it returns `0`instead of undefined. Let's ensure to improve upon this aspect.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Raphael, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁 🚦 🌴  🏃 🌳 🏡 🏘️ 🌲 

## Highlights 👍  
- Click events properly implemented!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/operate.js (hunk 1 above)
Comment (rmiyoyo) [reply src/logic/operate.js:20]: Hi, that's not my code? It came bundled with the project.

**Comment by rmiyoyo** [comment]
# Work Done
- Improved the operations

**Review by topeogunleye — APPROVED** [review]
### Hi @rmiyoyo, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Fixed errors encountered when dividing 0 by 0. ✔️ 
 
## [Optional] suggestions:
- For future pull requests, please notice that your branch name `events` does not follow the convention of being named after a feature or action. For example, you could name your branch `added-events`. This makes it easier to track the changes and the purpose of each branch.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**